packages/core/src/ember-html.ts

```diff
@@ -27,6 +27,11 @@ export interface EmberHTML {
 }
 
 class Placeholder {
+  static find(node: Node): Placeholder {
+    let placeholder = this.immediatelyAfter(node);
+    return placeholder;
+  }
```

Comment:
This function is the same as `replacing`, except that it doesn't remove the `node`. 
Before, the `replacing` function was used to replace the script `assets/vendor.js` with `@embroider/core/vendor.js`. Now, it simply finds the script `@embroider/core/vendor.js`, it doesn't remove it, and it uses its position as a reference for `insertAfter`. 

The reason I need all of this is because Fastboot scripts are still inserted by the compat-app-builder at this point, and we need to have the location of the vendor script to insert them at the correct position:
```
// compat-app-builder.ts

if (this.fastbootConfig) {
      // any extra fastboot vendor files get inserted into our
      // html.implicitScripts section, after the regular implicit script
      // (vendor.js) have been inserted.
      for (let script of this.fastbootConfig.extraVendorFiles) {
        html.insertScriptTag(html.implicitScripts, script, { tag: 'fastboot-script' });
      }
}
```